### PR TITLE
Js hangout categories

### PR DESCRIPTION
**Pull request by Kirazuto7**
Changes:

CardDeck.js: Properly shuffles deck on first load. Passes parameters of hangout categories and location to api fetch function for more dynamic fetching. The hardcoded values will be replaced by user's preferences in the future.
Saved swipes are organized by category in the database.

HangoutService.js: Added a category to each data retrieved based on the search term to later sort favorited places. Also made the code more dynamic and concise. Requires a location and passed in array of categories to fetch data.

### CODE REVIEW

File: my-app/src/Components/CardDeck.js
Comment (chizy7) [inline my-app/src/Components/CardDeck.js:1]: Changes here looks good. I noticed you addressed the workflow error from the previous merged code. Looks good.

File: my-app/src/Services/HangoutService.js
Comment (chizy7) [inline my-app/src/Services/HangoutService.js:1]: This looks great too. I see you made updates from the previous suggestions. I don't have any further suggestions. 
This looks great and I think it is ready to be merged after review from other reviewers.

**Review by chizy7 — APPROVED** [review]
Looks good. Thank you for making the updates.

**Comment by chizy7** [comment]
I just noticed the workflow error after my review. That should be addressed first.

**Review by caid4134 — APPROVED** [review]
The error has been addressed now (line 61's "==" to "==="). We likely need another PR for re-deployment though for the workflow.

**Review by JChen0915 — CHANGES_REQUESTED** [review]
The changes to the code look good, but I hesitate to approve this when it does not deploy on github actions. Though this is simply the bot being very picky about usage of syntax in this case, if deployment fails it cannot be approved

**Review by JChen0915 — APPROVED** [review]
Followup to previous review, with more communication Jordan has established this is an issue with previews that has been fixed in a commit already, approved

**Pull request merged**